.github/workflows/llm_integration.yml

```diff
@@ -456,9 +460,10 @@ jobs:
           sudo rm -rf models
           python3 llm/prepare.py fastertransformer_raw_aot t5-small
           ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
-          partition --model-dir /opt/ml/model/test/
+          train
```

Comment:
It looks like we support both `train` and `partition` as commands that invoke the partitioning. 

I'm assuming `train` is needed for SageMaker. Once we add SageMaker tests, can we keep the command consistent here?